CMakeLists.txt

```diff
@@ -49,10 +51,22 @@ set(HDBPP_ES_LIBRARIES)
 set(PKG_CONFIG_USE_CMAKE_PREFIX_PATH ON)
 list(APPEND CMAKE_PREFIX_PATH "/usr")
 
+# Set backends uri
+if(FETCH_LIBHDBPP_BACKEND STREQUAL "timescaledb")
```

Comment:
I find sometimes confusing that LIBHDBPP_BACKEND is "timescaledb", while the project is libhdbpp-timescale. Should we change the backend target to "timescale" ?

Reply:
Agreed and done. It is confusing enough as it is !